### DISCUSSION TITLE
真假有影？蘋果摺疊機傳明年有譜 雙螢幕、雙鏡頭「iPhone15 Flip」概念影片曝光

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#899] [General]
記得不久前才剛推出iphone14，現在竟然要推iphone15了!聽說是史上最貴的iphone，但我覺得要買一隻這樣的手機我還不如去買一台電腦呢==

https://tw.news.yahoo.com/%E7%9C%9F%E5%81%87%E6%9C%89%E5%BD%B1%EF%BC%9F%E8%98%8B%E6%9E%9C%E6%91%BA%E7%96%8A%E6%A9%9F%E5%82%B3%E6%98%8E%E5%B9%B4%E6%9C%89%E8%AD%9C-%E9%9B%99%E8%9E%A2%E5%B9%95%E3%80%81%E9%9B%99%E9%8F%A1%E9%A0%AD%E3%80%8Ci-phone-15-flip%E3%80%8D%E6%A6%82%E5%BF%B5%E5%BD%B1%E7%89%87%E6%9B%9D%E5%85%89-034313906.html

**Comment by HuangHsinTzu** [comment]
科技產業的進度真的太快了!
摺疊機真的很讚欸
小巧好攜帶 用來拍照也很方便
而且iphone還強調不會有折疊痕跡?!
只是
需要不少💸💸阿

**Comment by dabbie109303501** [comment]
雖然摺疊手機是很酷啦...
但我個人還是覺得摺疊手機好像沒必要?
感覺就只是手機加支架而已
而且花的錢是支架的好幾倍
不過這的確是個有趣的科技

**Comment by CPCpyt** [comment]
我很喜歡摺疊手機的概念，不握感覺滿損壞的，
希望可以先看其他人購買心得在覺得是否進場。

**Comment by StevenWanglolz** [comment]
我覺得這些概念機看看就好，每一代iphone推出後馬上就會有對下一代的猜測。不過希望蘋果下一代可以好好彌補一下去年的產品，ipad10根本是來搞耍的

**Comment by Doreen331** [comment]
之前有聽過一個說法是科技業的趨勢會不斷地輪迴，一聽到的時候覺得有一點困惑，但現在看到十年前的折疊式手機在被淘汰後，近期又以更進步的科技回歸，似乎開始了解這句話的意思了！

**Comment by Evan0979** [comment]
摺疊機目感覺技術還沒有到非常成熟,目前只是個噱頭吸引有錢人買

**Comment by Floriaand** [comment]
我之前一直看到三星的摺疊機開開合合之後，螢幕就不會亮了的案例，所以對折疊機沒有什麼好感。

**Comment by WEI-CHEN-CHUN** [comment]
科技的進步來源於人性，不過這類報導通常真實性待驗證

**Comment by Valentinaa0214** [comment]
希望也能出沒有摺疊功能的iphone15，
我覺得摺疊手機的設計真的好醜....，有點get不到它的美感哈哈
但若是只升級硬體的設計我還是會買單的
